cogs/Misc.py

```diff
@@ -100,10 +119,10 @@ def dump_tables_to_csv(tables: list[Table]) -> Iterable[discord.File]:
             with io.StringIO() as f:
                 writer = csv.writer(f)
                 writer.writerow(table.column_names)
-                writer.writerows(cursor.fetchall())
+                writer.writerows(cursor)
```

Comment:
`.fetchall()` is unnecessary here because `writerows()` iterates over the interable regardless. `fetchall` just requires us to store the entire list in memory at once, which is kinda wasteful.